benedict/serializers/cli.py

```diff
@@ -0,0 +1,81 @@
+from argparse import ArgumentError, ArgumentParser
+from collections import Counter
+from re import finditer
+
+from benedict.serializers.abstract import AbstractSerializer
+from benedict.utils import type_util
+
+regex_keys_with_values = r"-+\w+(?=\s[^\s-])"
+regex_all_keys = r"-+\w+"
+
+
+def parse_keys(regex, string):
```

Comment:
Maybe this could be a static method of `CLISerializer`

Reply:
Yeah that'd make more sense! I also added the regex_ strings into the class, but if you prefer they *do* remain outside feel free to (ask me to) change it